roles/edpm_neutron_sriov/molecule/default/converge.yml

```diff
@@ -16,7 +16,10 @@
 
 - name: Converge
   hosts: all
+  gather_facts: true
+  become: true
```

Comment:
no... 
this is still wrong you keep elevating to root globally

the role should eleve at the task level
if you need to od it here then there is a bug find the role.